ClangSharp.PInvokeGenerator/Diagnostic.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace ClangSharp
+{
+    public sealed class Diagnostic
```

Comment:
Diagnostics are really simple right now, but serve their purpose of reporting useful information well enough